WSL/toc.yml

```diff
@@ -4,18 +4,22 @@
   items:
     - name: What is WSL?
       href: about.md
+    - name: Best practices for set up
+      href: setup/environment.md
     - name: Comparing WSL 2 and WSL 1
       href: compare-versions.md
-- name: Quickstart
+- name: Install
   items:
-    - name: Install WSL & update to WSL 2
-      href: install-win10.md
-    - name: Best practices for setting up
-      href: setup/environment.md
+    - name: Install command
+      href: install-command.md
+    - name: Manual install
+      href: install-manual.md
     - name: Install on Windows Server
       href: install-on-server.md
     - name: Create a user account & password
```

Comment:
I think that these ones:
* Create a user account and password
* Import any Linux distribution

Would be better in "How to" or other sections since they aren't about installing WSL more about setting it up once it's installed.

Reply:
Moved create user account steps into the main install page and removed it as a separate page - please let me know what you think. I moved Import to the How-to section along with the create your own distro. I think that makes sense.